test/example.py

```diff
@@ -22,5 +22,7 @@ def factorial(n):
 age = int(input("Enter your age: "))
 if age >= 18:
     print("You are elligible for an adult privilege.") # typo
+    print("You are elligible for an adult privilege.") # typo
+    print("You are elligible for an adult privilege.") # typo
```

Comment:
```suggestion
    print("You are eligible for an adult privilege.") # typo
```
Please check this typo. Replace `elligible` with `eligible`.